#CNV-43159_how_to_detect (#93637)
openshift/openshift-docs: virt/managing_vms/virt-managing-vms-openshift-pipelines.adoc

include::modules/virt-running-ssp-pipeline-cli.adoc[leveloffset=+2]

[id="virt-deprecated-tasks_{context}"]
== Removing deprecated or unused resources

* You can clean up Tekton-Tasks-Operator resources associated with the operator.

.Procedure

. To remove any remaining Tekton-related resources from the cluster, run the following command:

> 

Reviewer: Add a plus sign here so that the code block appears indented correctly under Step 1.

include::modules/virt-running-ssp-pipeline-cli.adoc[leveloffset=+2]

[id="virt-deprecated-tasks_{context}"]
== Removing deprecated or unused resources

> You can clean up Tekton-Tasks-Operator resources associated with the operator.

Reviewer 2: ```suggestion
You can clean up deprecated or unused resources associated with the {pipelines-title} Operator.
```
A few comments:

- Where possible, use the attributes for product names. See https://redhat-documentation.github.io/supplementary-style-guide/#_product_names_and_version_references
- "Operator" must always be capitalized, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization
- Suggested slight edit to explain _why_ someone would remove the resources (they're deprecated or unused). The abstract / introduction sentence should explain why someone would do a procedure.

.Procedure

> * To remove any remaining Tekton-related resources from the cluster, run the following command:

Reviewer 2: ```suggestion
* Remove any remaining {pipelines-shortname} resources from the cluster by running the following command:
```
Notes
- I would recommend using the short name attribute for OpenShift Pipelines here. Tekton is the upstream project that the components are based on, but Pipelines is the product name, and seems to be used throughout this assembly.
- Suggested editing for minimalism and to make the step more active

+
[source,terminal]
----

> oc delete clusterroles,rolebindings,serviceaccounts,configmaps,pipelines,tasks \

Reviewer 2: ```suggestion
$ oc delete clusterroles,rolebindings,serviceaccounts,configmaps,pipelines,tasks \
```
Code blocks should start with a `$`. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

  --selector 'app.kubernetes.io/managed-by=ssp-operator' \
  --selector 'app.kubernetes.io/component in (tektonPipelines,tektonTasks)' \
  --selector 'app.kubernetes.io/name in (tekton-pipelines,tekton-tasks)' \
  --ignore-not-found \
  --all-namespaces
----
+

> If the Tekton tasks operator custom resource definitions (CRDs) have already been removed, the command may return an error. You can safely ignore this, as all other matching resources will still be deleted.

Reviewer 2: ```suggestion
If the {pipelines-title} Operator custom resource definitions (CRDs) have already been removed, the command may return an error. You can safely ignore this, as all other matching resources will still be deleted.
```
Same as above, use the literal Operator name and capitalize.
